chance/p5js/chance-similartiy/index.md

```diff
@@ -1,11 +1,11 @@
 ---  
 layout: p5js
-author: [josues]
-author-url: [https://incom.org/profil/7029]
-title: [Law of Similarity and chance]
-tags: [P5.js,[CHANCE], [SIMILARITY], Law of Similarity]
-permalink: /[CHANCE]/[chance-similarity]/
-summary: [This creates random similarities between ellipses by varying their size and color.]
+author: josues
+author-url: https://incom.org/profil/7029
+title: Law of similarity and chance
+tags: P5.js, chance, similarity, Law of similarity
+permalink: /chance/chance-similarity/
+summary: This creates random similarities between ellipses by varying their size and color.
 ---  
 
-![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-similarity/out.png)  
+![](https://raw.githubusercontent.com/josues/gestalten-in-code/master/chance/p5js/chance-similartiy/out.png)  
```

Comment:
Please no hardcoded urls,  If you want to refer to an image next to yours relativ markdown syntax  


    ![alt text](./out.png)  

But the code from your index.js will be integrated into the final site.